### ISSUE TITLE
event trigger: allow param function for dynamic values

### ISSUE THREAD

**Issue by PatrikTheDev** [ComfortablyCoding/strapi-plugin-website-builder#15]
Would it somehow be possible to send the document id or some field as a parameter to the request if the build was triggered by an event? This would make my revalidation code a lot easier and more efficient (I wouldn't have to check every single page for changes)

**Comment by ComfortablyCoding** [comment]
This is already possible via the `trigger.events.params` object, a sample configuration can be found [in the original feature request](https://github.com/ComfortablyCoding/strapi-plugin-website-builder/discussions/9#discussioncomment-2258983)

**Comment by PatrikTheDev** [comment]
This doesn’t necessarily solve my particular issue, I’d need the params to get the value of the model being edited.

proposed solution:

``` JavaScript 
module.exports = ({ env }) => ({
  "website-builder": {
    enabled: true,
    config: {
      url: "http://localhost:3000/api/revalidate",
      trigger: {
        type: "event",
        events: [
          {
            params: (page) => ({
              page: page.attributes.slug,
            }),
            model: "pages",
            types: ["create", "delete"],
          },
        ],
      },
    },
  },
});```

**Comment by matepaiva** [comment]
This is very handy! I would like to have it too.

**Comment by ComfortablyCoding** [comment]
This will be included in the next release

**Comment by ComfortablyCoding** [comment]
Released in v2.2.0

**Comment by matepaiva** [comment]
Thank you!
